crates/store/re_chunk_store/tests/snapshots/memory_test__scalars_on_one_timeline_new.snap

```diff
@@ -4,6 +4,6 @@ expression: "[format!(\"{NUM_SCALARS} scalars\"),\n        format!(\"{} in total
 ---
 [
     "1048576 scalars",
-    "37.5 MiB in total",
-    "38 B per row",
+    "37.0 MiB in total",
+    "37 B per row",
```

Comment:
.. hurray...?
So not sorting in some places saved memory? Guess something randomly compacted better?